gravitee-am-service/src/main/java/io/gravitee/am/service/impl/ReporterServiceImpl.java

```diff
@@ -378,7 +377,7 @@ public String createReporterConfig(String domain) {
 
     private String addOptionsToURI(Environment environment, String mongoUri) {
         Integer connectTimeout = environment.getProperty("management.mongodb.connectTimeout", Integer.class, 1000);
-        Integer socketTimeout = environment.getProperty("management.mongodb.socketTimeout", Integer.class, 1000);
+        Integer socketTimeout = environment.getProperty("management.mongodb.socketTimeout", Integer.class, 29000);
```

Comment:
Why is the value at `29000` ?

Reply:
The value has to be lower than HTTP timeout (30s)